### DISCUSSION TITLE
split rtl style from ltr style and lazy load them only when needed , reduce size and improve loading performance

### DISCUSSION THREAD

**Discussion by AlaaDq** [quasarframework/quasar#12525] [Ideas / Proposals]
split rtl style from ltr style and lazy load them only when needed , reduce size and improve loading performance.

**Comment by AlaaDq** [comment]
@MilosPaunovic 
can you help with this please?

**Comment by MilosPaunovic** [comment]
When I compare sizes of built `vendor.css` files with and without the `rtl` support enabled, there is a difference. So, Quasar does not add `rtl` styles by default. Do you have it turned on?

**Reply by AlaaDq** [reply]
Yes I am using it

**Reply by AlaaDq** [reply]
But I am wondering if there is a way to lazy load the required one only at a time

**Reply by MilosPaunovic** [reply]
You can split CSS code to smaller chunks by writing per component based styles, if one big file gives you problems with load times.

**Reply by AlaaDq** [reply]
I need split the vendor rtl from vendor ltr css styles.

**Reply by MilosPaunovic** [reply]
Quasar does not provide that option, at the moment.

**Reply by AlaaDq** [reply]
OK thanks
